### DISCUSSION TITLE
建议：丰富页面的自定义、增强配置的热修改

### DISCUSSION THREAD

**Discussion by soft98-top** [gaowei-space/markdown-blog#25] [Ideas]
场景：页面收录和seo优化
建议：
- footer内容丰富，友情链接、备案、Powered by等
- 自动生成sitemap，robots.txt
- 切换文章时，网站title变为文章名称
- 如果能自定义主题文件就更好了，但是又感觉程序会变得很复杂
- 配置的热修改，现在修改配置或者升级都是需要自己手动将进程关了再重启，热修改和热升级这边建议考虑一下。

**Comment by gaowei-space** [comment]
> 场景：页面收录和seo优化 建议：
> 
> * footer内容丰富，友情链接、备案、Powered by等
> * 自动生成sitemap，robots.txt
> * 切换文章时，网站title变为文章名称
> * 如果能自定义主题文件就更好了，但是又感觉程序会变得很复杂
> * 配置的热修改，现在修改配置或者升级都是需要自己手动将进程关了再重启，热修改和热升级这边建议考虑一下。

感谢反馈的建议，我这边会看下，逐步迭代一下
